net-p2p/resilio-sync/resilio-sync-2.6.1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BASE_URI="http://linux-packages.resilio.com/${PN}/deb/pool/non-free/r/${PN}/${PN}_${PV}-1_@arch@.deb"
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles unpacker user
+
+NAME="rslsync"
```

Comment:
Please don't. This makes figuring out the ebuild PITA.

Reply:
Do not use variables and use rslsync instead?

Reply:
Yes.